src/Server/Services/Http/CrdCrudControllerBase.cs

```diff
@@ -221,7 +231,33 @@ private object CreateCrdFromType(T item)
             throw new ApplicationException($"Unsupported data type: {item.GetType()}");
         }
 
-        
+        private void CheckAeTitleAndName(DestinationApplicationEntity destAe)
+        {
+            Guard.Again.Null(destAe, nameof(destAe));
+
+            if (string.IsNullOrWhiteSpace(destAe.Name) && destAe.AeTitle.Contains("_"))
+            {
+                throw new Exception("Please specified the `name` for the DICOM destination when AE Title contains underscore (_).");
```

Comment:
Suggest change to: "A `name` must be specified for the DICOM destination when AE Title contains underscore (_)."